test/distrib/bazel/run_bazel_distrib_test.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Copyright 2021 The gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+shopt -s nullglob
+
+cd "$(dirname "$0")"
+
+SUPPORTED_VERSIONS=(
+  "1.2.1"
+  "2.2.0"
+  "3.7.2"
+  "4.0.0"
+)
+
+for VERSION in "${SUPPORTED_VERSIONS[@]}"; do
+    ./test_single_bazel_version.sh "$VERSION"
```

Comment:
nit: the output of 4 consecutive builds is going to be difficult to parse by humans.
Ideally these would be 4 independent "tasks".

Reply:
Should we instead add 4-5  tasks into distribtest_targets.py  and save on needing to define 2 new kokoro jobs?
That would also make the output of the separate builds separated.